File: languages/zig/practice/control_flow/for.zig
const std = @import("std");
const print = std.debug.print;

pub fn main() !void {
    const friends [_][]u8{"Briana", "Elan", "Holden", "Reem"};

    for (friends) |friend| {
        if (friend == "Briana") {
            print("Briana is my sister and firend.");
        } else if (friend == "Reem") {
            print("Reem was my orientation leader and now my friend!");
        } else {
            print("{} is my friend.",.{friend});
        }
    }
}

File: languages/rust/practice/control_flow/for.rs
fn main() {
    let friends = ["Briana", "Elan", "Holden", "Reem"];

    for friend in friends {
        if friend == "Briana" {
            println!("Briana is my sister and firend.");
        } else if friend == "Reem" {
            println!("Reem was my orientation leader and now my friend!");
        } else {
            println!("{} is my friend.", friend);
        }
    }
}
